.github/dependabot.yml

```
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    commit-message:
+      prefix: ":seedling:"
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    commit-message:
+      prefix: ":seedling:"
```

You probably also want:
```suggestion
      prefix: ":seedling:"
    groups:
      k8s.io:
        patterns:
          - "k8s.io/*"
```

---

Thanks for the suggestion. We almost always manually bump k8s.io and controller-runtime, so I've instead added those to the ignore list.